### ISSUE TITLE
Can't download vips 7.42.3 when building the image

### ISSUE THREAD

**Issue by pixelastic** [netlify/build-image#92]
Hello,

When trying to build the image, I'm greeted with the following error:

```
gzip: stdin: not in gzip format
tar: Child returned status 1
tar: Error is not recoverable: exiting now
The command '/bin/sh -c curl -sO http://www.vips.ecs.soton.ac.uk/supported/$LIBVIPS_VERSION_MAJOR.$LIBVIPS_VERSION_MINOR/vips-$LIBVIPS_VERSION.tar.gz &&   tar zvxf vips-$LIBVIPS_VERSION.tar.gz &&   cd vips-$LIBVIPS_VERSION &&   ./configure --enable-debug=no --enable-docs=no --without-python --without-orc --without-fftw --without-gsf $1 &&   make &&   make install &&   ldconfig' returned a non-zero code: 2
```

By replacing the ENV variables, I tried to manually download the file (http://www.vips.ecs.soton.ac.uk/supported/7.42/vips-7.42.3.tar.gz), and ended up with a 404.

By checking the github, it also seems this version is not available anymore and I'm quite blocked on how to move forward.

Thanks

**Comment by fool** [comment]
Hmm, this isn't a problem with our build image, is it?  This sounds like you're trying to download a file from some external host that is not there - which isn't really something we can fix :)

If you can curl from your own machine but not from our build image that would be a different story - but I just tested curling a known-good path and it worked, so we'll need to see a repro case like that "this works locally and doesn't work on netlify" to continue.

**Comment by pixelastic** [comment]
Hmm, there is something in the reasoning I'm not following here. Maybe here is not the correct place to report the issue, so I apologize in advance if that's the case, but I still think there is an issue with the build image.

What I did was clone the repo and try to build your image locally. The image build could not complete because of the error I reported above. Specifically, those lines in the Docker file:
```Docker
WORKDIR /tmp
ENV LIBVIPS_VERSION_MAJOR 7
ENV LIBVIPS_VERSION_MINOR 42
ENV LIBVIPS_VERSION_PATCH 3
ENV LIBVIPS_VERSION $LIBVIPS_VERSION_MAJOR.$LIBVIPS_VERSION_MINOR.$LIBVIPS_VERSION_PATCH
RUN \
  curl -sO http://www.vips.ecs.soton.ac.uk/supported/$LIBVIPS_VERSION_MAJOR.$LIBVIPS_VERSION_MINOR/vips-$LIBVIPS_VERSION.tar.gz && \
  tar zvxf vips-$LIBVIPS_VERSION.tar.gz && \
  cd vips-$LIBVIPS_VERSION && \
  ./configure --enable-debug=no --enable-docs=no --without-python --without-orc --without-fftw --without-gsf $1 && \
  make && \
  make install && \
  ldconfig
```

As far as I understand, the build process, following the Dockerfile instructions, is trying to download a file and fails. When I manually try to download the same file (by manually replacing the ENV variables in the generated url), and testing locally, it fails as well.

So my issue is not related to `curl`ing from inside the image, but to actually building the image myself. I know you have a built image already available on `netlify/build`, but I'd like to build my own local version.

**Comment by fool** [comment]
OH!  I apologize @pixelastic - I thought you were saying "I can't use that command in a build of a netlify site" and I was thinking "ok, so change the command/path?"

That does seem to be a bug - we'll get a patch in place for it when we next rebuild the docker image.  In the meantime - you could chop the whole section out of the Dockerfile to get things to build (unless you need libvips yourself?)

**Comment by fool** [comment]
(Additional detail:

- libvips moved their downloads to GH pages, new URL to download from, but...
- ...that release does not seem to be available for download anymore. )

**Comment by pixelastic** [comment]
Oh I don't need `libvips`. Well, actually I don't even know what it's used for :) but I thought it would be a dependency for something else. If it's not needed, I'll remove it from the Dockerfile for my tests then.

Thank you :)

**Comment by fool** [comment]
Some people need it - but you probably don't :)

**Comment by brycekahle** [comment]
Fixed in cd9bbfa4c3013a3f59598c9bba820f5ab4056351
